models/boost/boost_claimed.sql

```diff
@@ -0,0 +1,102 @@
+{{
+    config(
+        unique_key='claimer_address',
```

Comment:
`unique_key` is only needed if you specify the model to be `incremental`

do you plan to materialize cross-chain spells? or are views for these fine too?

Reply:
again, refer to the [docs ](https://github.com/duneanalytics/spellbook/tree/main/docs/models) on models if you want to change materialization type and how to configure

Reply:
this would apply to all cross-chain spells in PR

Reply:
Thank you for this!! View may be too slow -- had to wait too long for the test views to load and it's not practical for opening this spell to public use. I made the cross-chain spells `incremental` instead.